example_school_library_decorator/student.rb

```diff
@@ -1,12 +1,19 @@
-require './person.rb'
+# rubocop:disable Style/OptionalBooleanParameter
+require './person'
```

Comment:
- [ ] Also sets the classroom for the student as the requirements say

![mlmlmlml](https://user-images.githubusercontent.com/61562626/135163405-2ce09206-1288-4885-9cb8-befd6ce00d87.jpeg)
.